plugins/woocommerce/src/StoreApi/Routes/V1/CartUpdateCustomer.php

```diff
@@ -132,26 +132,19 @@ protected function get_route_post_response( \WP_REST_Request $request ) {
 		$customer = wc()->customer;
 
 		// Get data from request object and merge with customer object, then sanitize.
-		$billing  = $this->schema->billing_address_schema->sanitize_callback(
-			wp_parse_args(
-				$request['billing_address'] ?? [],
-				$this->get_customer_billing_address( $customer )
-			),
-			$request,
-			'billing_address'
-		);
-		$shipping = $this->schema->billing_address_schema->sanitize_callback(
```

Comment:
Also this was the wrong schema to validate against, this doesn't cause any issue in the core, but it might cause issues if extensions adds fields only in shipping or billing